toolchest_client/tools/unicycler.py

```diff
@@ -23,4 +25,8 @@ def __init__(self, tool_args, output_name, inputs, input_prefix_mapping,
             input_prefix_mapping=input_prefix_mapping,
             min_inputs=1,
             max_inputs=3,
+            parallel_enabled=False,
+            output_type=OutputType.GZ_TAR,
+            output_is_directory=True,
+            output_names=["assembly.fasta"],
```

Comment:
Some tools – like Unicycler – have a lot more outputs by default. In the case of Unicycler, we have:

```
001_best_spades_graph.gfa 
003_bridges_applied.gfa 
005_polished.gfa 
assembly.gfa
r1.fastq 
unassembled.fasta
002_overlaps_removed.gfa  
004_final_clean.gfa
assembly.fasta
contigs.fasta.gz
r2.fastq 
unicycler.log
```

All of which I'm fine with keeping in the default outputs, but I'm not sure we want `output_names` to express a _complete_ overview of the output files; some output files change dependent on the inputs and args. Maybe we just try to grab primary output + log?